openlibrary/templates/account/ia_thirdparty_logins.html

```diff
@@ -1,12 +1,24 @@
-$def with ()
+$def with (lazy=False)
 
 $ params = {'origin': request.home.replace('http:', 'https:')}
 <iframe
     id="ia-third-party-logins"
-    src="https://archive.org/account/login.thirdparty.php?$(urlencode(params))"
+    $("src" if not lazy else "data-src")="https://archive.org/account/login.thirdparty.php?$(urlencode(params))"
 
-    style="border:0; width: 100%; height: 44px"
+    style="border:0; width: 100%; min-width: 250px; height: 40px"
 ></iframe>
+
+<script>
+    window.addEventListener('LOAD_THIRD_PARTY_LOGINS', function() {
+        var iframe = document.getElementById('ia-third-party-logins');
+        iframe.src = iframe.getAttribute('data-src')
+    });
+
+    window.addEventListener('UNLOAD_THIRD_PARTY_LOGINS', function() {
+        var iframe = document.getElementById('ia-third-party-logins');
+        iframe.src = '';
+    });

```

Comment:
Let's define these as globals:
```suggestion
    window.LOAD_THIRD_PARTY_LOGINS = function() {
        var iframe = document.getElementById('ia-third-party-logins');
        iframe.src = iframe.getAttribute('data-src');
    };

    window.UNLOAD_THIRD_PARTY_LOGINS = function() {
        var iframe = document.getElementById('ia-third-party-logins');
        iframe.src = '';
    };

```

Then call these functions directly with eg `window.LOAD_THIRD_PARTY_LOGINS()`. The event handling creates an extra level of misdireciton, and uses some things which would require polyfilling (eg `Event`)